quodlibet/qltk/songlist.py

```diff
@@ -161,7 +163,9 @@ def get_sort_tag(tag):
 
     if "<" in tag:
         for key, value in replace_order.items():
-            tag = tag.replace("<%s>" % key, "<%s>" % value)
+            if value != "":
+                value = "<%s>" % value
```

Comment:
```suggestion
                value = f"<{value}>"
```
Trying to convert to f-strings slowly everywhere